src/components/common/MenuAppBar.tsx

```diff
@@ -25,6 +25,7 @@ import { Colours } from '../../styles/Constants';
 interface MenuAppBarProps {
   eventId: string;
   pageTitle: string;
+  selectedCcp?: string;
```

Comment:
the new prop is optional, so it shouldn't affect current instances of the component